documentation/docs/aggregation.md

```diff
@@ -0,0 +1,37 @@
+# What is an aggregation formula?
+
+![Group 3](https://user-images.githubusercontent.com/66801011/224264480-a66d91be-fa04-4994-a719-d390d2297ff4.png)
+
+An aggregation formula is an arithmetic formula that decides how to aggregate the result of an arbitrary amount of sub-aggregations and metrics.
+
+Aggregating metric results is a way of approximating higher level attributes, such as maintainability or reliability. _Modu_ does this by letting the user define what sub-aggregations and metrics aggregate into these higher level attributes, and potentially their importance as well. 
```

Comment:
"potentially their importance as well" - what is meant by this?

Reply:
Yah.. this is still possible I guess as you can create the aggregations as you please. So maybe we should just mention the weight by name in a short sentence?

Reply:
I changed this and gave an example (weighted average)